setup-travis.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ev
+
+gem update --system
+gem --version
+#cache file containing vendor/bundle/cache - to speed up bundle instann and to fix possible connection itmeouts to rubygems
+wget -qO- http://testmycode.net/travis/2.2.0-tmc-server-bundler-cache.tar.gz | tar xz
+BUNDLE_GEMFILE=Gemfile bundle install --retry=3 --jobs=3 --deployment
+# Reduce mavens memory usage
+export MAVEN_OPTS="-Xms512m -Xmx1024m -XX:PermSize=1024m"
+# So that travis can clone submodules
+sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+git submodule update --init --recursive
+git config --global user.email "travis@example.com"
+git config --global user.name "Travis"
+  # tmc-check
+export CHECK_INSTALL_DIR=$HOME/check
+mkdir $CHECK_INSTALL_DIR
+git clone https://github.com/testmycode/tmc-check.git
+cd tmc-check
+env BUNDLE_GEMFILE=$(pwd)/Gemfile bundle install --jobs=3 --retry=3 --deployment
+env BUNDLE_GEMFILE=$(pwd)/Gemfile make
+env BUNDLE_GEMFILE=$(pwd)/Gemfile make install PREFIX=$CHECK_INSTALL_DIR BUNDLE_GEMFILE="$(pwd)/Gemfile"
+export PATH="$CHECK_INSTALL_DIR/bin:$PATH"
+cd ..
+export LD_LIBRARY_PATH=$CHECK_INSTALL_DIR/lib:/lib:/usr/lib:/usr/local/lib:$LD_LIBRARY_PATH
+export C_INCLUDE_PATH=$CHECK_INSTALL_DIR/include:$C_INCLUDE_PATH
+export PKG_CONFIG_PATH=$CHECK_INSTALL_DIR/lib/pkgconfig:$PKG_CONFIG_PATH
+  #tmc comet, compile manually since rake task keeps on failing in travis...
+cd ext/tmc-comet/
+mvn install -q -Dmaven.test.skip=true
+cd ../../
+  # Build submodule deps
+BUNDLE_GEMFILE=Gemfile bundle exec rake compile
+  # use prebuilt tmc-sandbox
+wget -qO- http://testmycode.net/travis/sandbox-$(git submodule status ext/tmc-sandbox | grep -E -o  "[0-9a-f]{40}").tar.gz | tar xvz -C ext/
+cd ext/tmc-sandbox/web
+sed -i "s/ tmc/ $(whoami)/" site.defaults.yml
+sed -i 's/max_instances: [0-9]/max_instances: 2/' site.defaults.yml
```

Comment:
`[0-9]*`? (or with [-r and +](http://stackoverflow.com/questions/22099623/sed-plus-sign-doesnt-work))